utils.go

```diff
@@ -90,19 +90,27 @@ func newStorager(pairs ...types.Pair) (store *Storage, err error) {
 	}
 
 	p := azfile.NewPipeline(credValue, azfile.PipelineOptions{
-		Retry: azfile.RetryOptions{
-			// Use a fixed back-off retry policy.
-			Policy: 1,
-			// A value of 1 means 1 try and no retries.
-			MaxTries: 1,
-			// Set a long enough timeout to adopt our timeout control.
-			// This value could be adjusted to context deadline if request context has a deadline set.
-			TryTimeout: 720 * time.Hour,
-		},
+		Retry: azfile.RetryOptions{},
 	})
 
+	serviceURL := azfile.NewServiceURL(*primaryURL, p)
+
+	ctx := context.Background()
+	shareURL := serviceURL.NewShareURL(opt.Name)
+	_, err = shareURL.Create(ctx, azfile.Metadata{}, 0)
```

Comment:
This will create a new shared, is it expected?

Reply:
Here you have to set the files to be shared in order to call all the next APIs.
If this share file exists, no new ones will be created.

Reply:
Oh, it doesn't make sense.

The `share` here is the same as `bucket` of `object storage services, I think we should return instead of creating one for the user.

Let's ask ourselves a question: What's the user expected?

Reply:
Oh, yes, `share` is like `bucket` here, if it doesn't exist it should return an error instead of creating one.
The user should expect that the share is already created.